src/saml.ts

```diff
@@ -470,12 +470,16 @@ class SAML {
     additionalParameters: querystring.ParsedUrlQuery
   ): Promise<string> {
     this.options.entryPoint = assertRequired(this.options.entryPoint, "entryPoint is required");
+    const requestOrResponse = assertRequired(
+      request || response,
+      "one of request and response is required"
```

Comment:
nit: "one of request _or_ response is required"

Reply:
Or better "Either request or response is required"

Reply:
fixed, thanks